### DISCUSSION TITLE
Is there a way to make python-format-on-save project dependent?

### DISCUSSION THREAD

**Discussion by gderber** [syl20bnr/spacemacs#16496] [Q&A] [answered]
I like python-format-on-save enabled.  It works great on my own projects.  However, when I am working on an outside project, it creates havoc.

Is there a way to set that based on the project I am working in?

**Comment by fnussbaum** [comment, marked as answer]
One way to achieve this is to put a .dir-locals.el file with an appropriate entry to enable it into the root of your own projects, see https://www.gnu.org/software/emacs/manual/html_node/emacs/Directory-Variables.html and <kbd>SPC f v d</kbd>.

**Comment by gderber** [comment]
Thank you!
